defaults/main.yml

```diff
@@ -167,7 +167,11 @@ nginx_http_template:
     template_file: http/default.conf.j2
     conf_file_name: default.conf
     conf_file_location: /etc/nginx/conf.d/
-    port: 8081
+    listen:
+      port: 8081
+      opts: []
```

Comment:
Can you include the comment from the README?

Reply:
@alessfg done :)